.github/workflows/release-please.yml

```diff
@@ -38,7 +29,28 @@ jobs:
       - name: Install dependencies
         run: npm ci
         if: ${{ steps.release.outputs.release_created }}
-      - run: npm publish packages/${{ matrix.package }}/
-        if: ${{ steps.release.outputs.release_created }}
+      # Publish new releases in order of dependencies
```

Comment:
When a release was created for a package `outputs` will have a corresponding `--version` property